assets/js/theme/common/product-card.js

```diff
@@ -0,0 +1,30 @@
+const focusedCardClass = 'focused-card';
+const focusableElementsSelector = '[href], button, input, textarea, select, details, [tabindex]';
+const previousCard = {
+    $previous: null,
+};
+
+const onCardFocus = (event) => {
+    const $element = $(event.target);
+
+    const $cardContainer = $element.parents('.card'); 
+
+    const isFocusInCard = $cardContainer.length;
+
+    const { $previous } = previousCard;
+    
+    if (isFocusInCard) {
+        $previous && $previous.removeClass(focusedCardClass); 
+        $cardContainer.addClass(focusedCardClass);
+        previousCard.$previous = $cardContainer;
+        return;
```

Comment:
I would be nice to remove 17 and 20 code lines

Reply:
> Generally, I think https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-within would be helpful in this situation (polyfill for IE will be needed)

Maybe I can add focus-within, detect IE and use my script in ie like polifill?

Reply:
> > Generally, I think https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-within would be helpful in this situation (polyfill for IE will be needed)
> 
> Maybe I can add focus-within, detect IE and use my script in ie like polifill?

test my code in ie 11 and it works

Reply:
I prefer https://www.npmjs.com/package/focus-within-polyfill